File: problem/Minimum_Changes/minOperations.java
class Solution {
    public int minOperations(String s) {
        int count = 0, n = s.length();
        for (int i = 0; i < n; i++)
            count += (s.charAt(i) ^ i) & 1;
          
        return Math.min(count, n - count);
    }
}

File: problem/Minimum_Changes/minOperations.js
/**
 * @param {string} s
 * @return {number}
 */
const minOperations = s => {
    let count = 0, n = s.length;
    for (let i = 0; i < s.length; i++)
        if (s[i] ^ (i & 1)) count++;

    return Math.min(count, n - count);
};